### ISSUE TITLE
Possibly Fixed? Unable to linear navigate to "Copy Link" button under downloads on ChromeOS

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1052]
Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: User unable to linear navigate to copy button under Share dialog_Web_1.3	"ENVIRONMENT:
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Turn on the respective screen reader

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3) Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Linear navigate to the share button on the page and perform action
5). Try linear navigating to 'Copy link to clipboard' button on the dialog
6). Observe the behavior

EXPECTED RESULT:
User should be able to linear navigate onto all elements on a page.
 
OBSERVED RESULT:
User is unable to linear navigate onto 'Copy link to clipboard' button on the dialog. User is able to reach the button using tab navigation.

Replication video attached"	https://drive.google.com/file/d/1f1AtmGcY8iweiGkQlgdGLJA_89fx7Hho/view?usp=sharing	Same issue is observed on Compare rates page.		ben- not observed with safari VO or firefox windows NVDA or chrome screen reader; I don't have access to chromevox on chromeOS so cant test

**Comment by benhammondmusic** [comment]
not observed with safari VO,\ firefox windows NVDA, nor chrome screen reader on chrome MacOS

I don't have access to chromevox on chromeOS so cant truly test but perhaps this has been fixed already?

**Comment by benhammondmusic** [comment]
feature no longer exists
